error.go

```diff
@@ -61,6 +68,13 @@ func (e *APIError) UnmarshalJSON(data []byte) (err error) {
 		}
 	}
 
+	if _, ok := rawMap["innererror"]; ok {
+		err = json.Unmarshal(rawMap["innererror"], &e.InnerError)
+		if err != nil {
+			return
+		}
+	}
```

Comment:
@ZeroDeng01 Please write the test code to cover the code above.

Reply:
Complete